crawler/crawler/crawler.go

```diff
@@ -144,7 +144,32 @@ func (c *Crawler) CrawlPublishers(publishers []PA) error {
 		close(c.repositories)
 	}()
 
-	return c.crawl()
+	// here we got all repos to be scanned
+	// it's time to check blacklist and wheter one of them
+	// is listed.
+	// we should return the ones listed to crawl command
+	// and call deleteFromES if present
+	toBeRemoved := c.removeBlackListedFromRepositories(GetAllBlackListedRepos())
+
+	return toBeRemoved, c.crawl()
+}
+
+func (c *Crawler) removeBlackListedFromRepositories(listedRepos map[string]string) (toBeRemoved []string) {
+	temp := make(chan Repository, 1000)
```

Comment:
I don't get this, could you please add a comment? Thank you

Reply:
Basically this function is in charge to assign a new channel that discards repositories in blacklists.